guides/command-implementation.md

```diff
@@ -26,3 +26,86 @@ have
    1. describe what this command is for (what it does)
    2. information that you think that users should be aware of
    3. sample(s) of the command
+
+## Guidelines
```

Comment:
I would suggest adding a link to this doc in `contributing.md`.

Reply:
accept